templates/wiki/wiki_top_navbar.html

```diff
@@ -0,0 +1,65 @@
+{% load static %}
+{% include 'common/common_fonts.html' %}
+
+<style>
+@font-face {
+    font-family: 'Manrope';
+    src: url("{% static 'fonts/Manrope-Light.ttf' %}");
+    font-weight: 300;
+}
+@font-face {
+    font-family: 'Manrope';
+    src: url("{% static 'fonts/Manrope-Regular.ttf' %}");
+    font-weight: 400;
+}
+@font-face {
+    font-family: 'Manrope';
+    src: url("{% static 'fonts/Manrope-Medium.ttf' %}");
+    font-weight: 500;
+}
+@font-face {
+    font-family: 'Manrope';
+    src: url("{% static 'fonts/Manrope-Bold.ttf' %}");
+    font-weight: bold;
+}
+
+hr { margin-bottom: 0; }
+.navbar { padding: 0; }
+.bg-dark, .bg-light { background-color: rgb(248, 248, 248) !important; }
+.nav-link { color: rgb(25, 25, 25); font-family: "Manrope", serif;     font-size: .825rem; line-height: 1.625; }
+.py-2 { margin-bottom: 0 !important; padding-bottom: 0 !important; }
+#navbarColor02 { background-color: rgb(33, 37, 41); padding: 16px; }
+</style>
```

Comment:
Is there a reason for this to not be a separate css file?

Reply:
Yes. We do not seem to have access to the html head section to insert a new css ref when overriding the django-wiki template. I think this would either require a PR to django-wiki for an a tag to override with custom css references or as they suggest using bootstrap scss. But I think that is a but outside the scope of this PR and worth discussing the approach with the team.

Reply:
i suggest we leave a link to this conversation as a comment
https://github.com/ScilifelabDataCentre/stackn/pull/41/files#r1254570218

Reply:
Good suggestion. Done.